notebooks/Periodicity-Detection/Periodicity_Detection.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<p>It might be interesting to also use cuXfilter to do the plots. Just a suggestion though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I had raised an issue on GH to ask for CuPy support for simple plots just like plotting a simple array with numpy+matplotlib. I'll check again.